src/main/java/org/jenkinsci/plugins/githubautostatus/notifiers/InfluxDbNotifierSchemas.java

```diff
@@ -61,7 +61,9 @@ public static SchemaInfo getSchema(int schemaIndex) {
         private static final String Result = "result";
         private static final String Blocked = "blocked"; // This is for v1 compat; don't use as a tag going forward
         private static final String StageName = "stagename";
+        @Deprecated
         private static final String Suite = "suite";
+        private static final String SuiteName = "suite";
```

Comment:
It's important to change the value sent to influxdb

```suggestion
        private static final String SuiteName = "suitename";
```